docs/fetching-credentials.md

```diff
@@ -0,0 +1,73 @@
+---
+id: fetching-credentials
+title: Fetching Credentials
+---
+
+As seen [here](https://support.demisto.com/hc/en-us/articles/115002567894), it is possible to integrate with 3rd party credential 
```

Comment:
Links to a page requiring authentication...not sure if intentional.

Reply:
I think it is ok. Dev should have access. Anyway, we can improve later on if needed.